Client-Side/shecodesprogramming/src/Components/HomePage/homepage.jsx

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import './homepage.css';
+
+function Homepage(){
+    return(
+        <div>
+            <div className="MainLayout">
+                <div className="layout">
+                    <div className="sublayout">
+                        <img className="welcome" src="https://kalvium.community/images/home_hero.gif" alt="" />
+                        <h2></h2>
+                        <p>Welcome to this amazing place where you can have lots of fun and learn new things! It's a special spot where you'll experience the joy of learning and discover the wonders of magic. Enjoy every moment of your journey here!</p>
+                        <p>To begin learning the courses, simply go to Livebooks. It's a platform where you can access educational materials and resources. It's like a virtual library for learning. Once you're there, you'll find a variety of courses and materials to explore and study. It's a great place to start your learning journey and access the knowledge you need. Enjoy your learning experience on Livebooks!</p>
+                    </div>
+
+                    <div className="display">
+                        <div>
+                            <img className="whyimage" src="https://uploads-ssl.webflow.com/620ce06fa12cad2d27fb30a8/630cc6df6db69f0f8c9f313a_1%20flipped-p-800.png" alt="" />
```

Comment:
Please add alternate text for image tag

Reply:
I added text for alternate could you please review it.